index.html

```diff
@@ -5267,17 +5318,17 @@ <h3>Algorithm</h3>
                   to a JSON <a>number</a>.</li>
               </ol>
             </li>
-            <li class="changed">Otherwise, if <a>processing mode</a> is <code>json-ld-1.1</code>,
+            <li class="changed">Otherwise, if <a>processing mode</a> is not `json-ld-1.0`,
               and <var>value</var> is a <a>JSON literal</a>,
               set <var>converted value</var> to the result of
               turning the lexical value of <var>value</var>
               into the <a>JSON-LD internal representation</a>, and set <var>type</var> to <code>@json</code>.
+              Set <a>processing mode</a>, to `json-ld-1.1`, if not already set.
               If the lexical value of <var>value</var> is not valid JSON according to
               the <a data-cite="RFC8259#section-2">JSON Grammar</a> [[RFC8259]],
               an <a data-link-for="JsonLdErrorCode">invalid JSON literal</a>
               error has been detected and processing is aborted.</li>
-            <li class="changed">Otherwise, if <a>processing mode</a> is `json-ld-1.1`,
-              the <a>datatype IRI</a> of <var>value</var> starts with `https://www.w3.org/ns/i18n#`,
+            <li class="changed">Otherwise, if the <a>datatype IRI</a> of <var>value</var> starts with `https://www.w3.org/ns/i18n#`,
```

Comment:
If processing mode is explicitly json-ld-1.0, I would expect this kind of datatype to be kept as is...
So the condition would become
```
if <a>processing mode</a> is not `json-ld-1.0` and the <a>datatype IRI</a>...
```

Reply:
It shouldn't be necessary, as the statement is already predicated on the `rdfDirection` option, which an only appears in 1.1.